rclcpp/include/rclcpp/node.hpp

```diff
@@ -866,6 +866,29 @@ class Node : public std::enable_shared_from_this<Node>
    * When a callback returns a not successful result, the remaining callbacks aren't called.
    * The order of the callback is the reverse from the registration order.
    *
+   * It is considered bad practice to reject changes for "unknown" parameters as this prevents
+   * other parts of the node (that may be aware of these parameters) from handling them.
```

Comment:
It might be a good idea to just move this part of the note up above (where the initial parameter callback example is), and just drop the "do not do this" code.  That will shorten this already lengthy comment, and not put bad code in here that people will blindly copy-n-paste.

Reply:
See a056e3c